File: Lógica de Programação Java e C#/5 - Arrays/C#/Exercício de Fixação/TabelaQualquer.cs
class TabelaQualquer
{
    static void Main()
    {
        int[][] matriz = new int [3][];
        for (int a = 0; a < matriz.Length; a++)
        {
            matriz[a] = new int[3];
        }
        matriz[0][0] = 12;
        matriz[0][1]=15;
        matriz[0][2]=18;
        matriz[1][0]=21;
        matriz[1][1]=25;
        matriz[1][2]=31;
        matriz[2][0]=43;
        matriz[2][1]=56;
        matriz[2][2]=78;
        for(int a=0;a<matriz.Length;a++){
            for(int b=0;b<matriz[a].Length;b++){
                System.Console.WriteLine(matriz[a][b]);
            }
        }
    }
}

File: Lógica de Programação Java e C#/5 - Arrays/Java/Exercícios de Fixação/TabelaQualquer.java
class TabelaQualquer{
	public static void main(String [] args){
		int [][] array = new int [3][3];
			array[0][0]= 19;
			array[0][1] = 22;
			array[0][2] = 31;
			array[1][0] = 2;
			array[1][1] = 51;
			array[1][2] = 12;
			array[2][0] = 65;
			array[2][1] = 40;
			array[2][2] = 43;
		
		for(int a=0;a<array.length;a++){
			for(int b=0;b<array[a].length;b++){
				System.out.println(array[a][b]);
			}
		}
	}
}
	